app/models/generic_object.rb

```diff
@@ -95,6 +103,28 @@ def property_setter(name, value)
     self.properties = properties.merge(name => val)
   end
 
```

Comment:
@lfu @gmcculloug @bzwei 
Should we have a method instance_methods  defined for generic objects which will produce a list of known methods supported by the instance. This might come in handy for REST API so that it can list the available methods and the parameters, the missing_method won't help the REST API when it needs to define list of available methods.